tests/e2e/pact/postman-pact.ts

```diff
@@ -53,5 +59,7 @@ function createPostmanItem(interaction) {
         item.request.header.push({key, value})
     }
 
+    item.request.header.push({key: "Authorization", value: `Bearer ${process.env.APIGEE_ACCESS_TOKEN}`})
```

Comment:
Not sure how much use this is given it'll expire after 10 minutes, but I guess it shows you where to put the token?

Reply:
Not amazingly useful, but more useful than not having a token, if you run the tools to generate new prescriptions and you want to send a few into int then you can do that easily enough within the time, also gets overidden if you set postman authorization header so as a default it's ok